### DISCUSSION TITLE
GitHub Sync Timing Out

### DISCUSSION THREAD

**Discussion by EvolvingSysadmin** [GitbookIO/community#741] [3. Git Sync]
Am I the only one experiencing this issue? The sync service worked fine yesterday. Now I'm unable to sync in either direction for any of my spaces, even after uninstalling/reinstalling the integration/app in both GitBook and GitHub. Of course, the timing of this issue is particularly bad because of impending deadlines I must meet in 2 days, so I'll likely have to use a different platform.

**Comment by junxu-ai** [comment]
i have the same issue.

**Comment by SpeedKRNL** [comment]
I have the same problem

**Comment by damonYuan** [comment]
same problem here

**Comment by 0xChicharito** [comment]
i also can not synchronize

**Comment by macervero-abello** [comment]
Same problem here :(

**Comment by addisonschultz** [comment]
Hi all - we're immediately looking into this issue, thank you for your patience!

**Comment by luisgarciaxygeni** [comment]
Same problem. no advance ..
<img width="411" alt="image" src="https://github.com/user-attachments/assets/a9301d4e-3e49-4026-839d-be6a727d6156">

**Comment by CharlesShan-hub** [comment]
Same problem！Neither my old notebook nor a new-build notebook with vary little content，cannot sync to my github

**Comment by macervero-abello** [comment]
Problem solved!! Thank you very much, Gitbook Team!!

**Comment by aikoii** [comment]
Hey everyone! Thank you for your patience while our engineering team figured this one out.
​
The issue has been resolved, and Git Sync should now be fully functional again 🤞.

**If your last change before the error was made on GitBook** - meaning you had a change request that you merged and got an error from Git Sync - please create another change request with any small change in your documentation and merge it. That should export all the previous changes and restore sync.

**If your last change before the error was made on your repo** - meaning you made a commit that didn't get imported to GitBook and received an error - please create another commit with a small change in your repo. That commit should be picked up by GitBook automatically, and imported correctly alongside all the previous changes before the error and restore sync.
​
Feel free to reach out to support@gitbook.com if anything comes up.

Thank You!
